### ISSUE TITLE
Normalizer

### ISSUE THREAD

**Issue by eliecharra** [kitpages/KitpagesDataGridBundle#46]
Version : **v2.4.0**

When using a group by, item normalizer create a sub array for associated objects.

```php
$queryBuilder->select('offer, partner, count(s.id) as subscriptionCount')
    	->from('Offer', 'offer')
    	->innerJoin('offer.partner', 'partner')
        ->leftJoin('offer.subscriptionList', 's', Expr\Join::WITH, 's.offer = offer.id')
    	->groupBy('offer.id');
```

**Actual :**
```php
  "offer.id" => 1572
  "offer.partner" => array:8 [
    "name" => "Test Partner"
    "id" => 68
  ]
```

**Expected :**
```php
  "offer.id" => 1572
  "partner.name" => "Test Partner",
  "partner.id" => 68
  ]
```

**Comment by eliecharra** [comment]
PR #47

**Comment by eliecharra** [comment]
Released as [v2.4.1](/kitpages/KitpagesDataGridBundle/releases/tag/v2.4.1)
